hack/make.ps1

```diff
@@ -437,6 +437,7 @@ Try {
 }
 Catch [Exception] {
     Write-Host -ForegroundColor Red ("`nERROR: make.ps1 failed:`n$_")
+    Write-Host -ForegroundColor Red ($_.InvocationInfo.PositionMessage)
```

Comment:
Interesting! How to use this against, e.g, a running container process?

The `DockerSwarmSuite.TestAPISwarmLeaderElection()` should be a flaky one.